src/Templates/src/templates/maui-mobile/PageModels/ProjectDetailPageModel.cs

```diff
@@ -177,6 +180,18 @@ private async Task TaskCompleted(ProjectTask task)
 		OnPropertyChanged(nameof(HasCompletedTasks));
 	}
 
+	partial void  OnIsCategoryPickerExpandedChanged(bool value)
+	{
+		if (value)
+		{
+			SemanticScreenReader.Announce("State Expanded");
```

Comment:
The current announcements "State Expanded" and "State Collapsed" lack context about what was expanded/collapsed.